samples/secure-mcp-sse-client-server/secure-mcp-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+quarkus.langchain4j.mcp.generate-tool-provider.transport-type=http
```

Comment:
See the same comment that I made at https://github.com/quarkusio/quarkus-workshop-langchain4j/pull/73#discussion_r2083916912

Where did you get the name `generate-tool-provider` from? The fact that two people did this mistake at the same time makes me think that there is something wrong with our documentation or some other samples.

Reply:
@jmartisk I think I copied it from the `mcp-sse-client-server/mcp-client` :-), sure,  I'll update

Reply:
I see, ok, I'll change it there, thanks :)

Reply:
I can update that demo too along the way